### PR TITLE
C360553 Verify that user barcodes shown in request actions (Volaris) (TaaS)

### PR DESCRIPTION
**Pull request by victoriakor**
https://foliotest.testrail.io/index.php?/cases/view/360553
https://jenkins-aws.indexdata.com/job/folioRancher/job/folioTestingTools/job/runCypressTests/5023/allure/

### CODE REVIEW

File: cypress/e2e/circulation-log/verify-user-barcode-shown-in-request-actions.cy.js [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+      testData.defaultLocation.libraryId,
+      testData.defaultLocation.id,
+    );
+  });
+
+  it(
+    'C360553 Verify that user barcodes shown in request actions (Volaris) (TaaS)',
+    { tags: [TestTypes.extendedPath, DevTeams.volaris] },
+    () => {
+      cy.login(userData.username, userData.password, {
```
Comment (dnovikau) [inline cypress/e2e/circulation-log/verify-user-barcode-shown-in-request-actions.cy.js:80]: Please move 'cy.login' to 'cy.createTempUser' in preconditions

File: cypress/e2e/circulation-log/verify-user-barcode-shown-in-request-actions.cy.js [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+        path: TopMenu.requestsPath,
+        waiter: RequestsSearchResultsPane.waitLoading,
+      });
+      // Create new request with item barcode anf requester barcode
+      cy.visit(TopMenu.requestsPath);
```
Comment (dnovikau) [inline cypress/e2e/circulation-log/verify-user-barcode-shown-in-request-actions.cy.js:85]: Please remove this

File: cypress/e2e/circulation-log/verify-user-barcode-shown-in-request-actions.cy.js [hunk 3]
```diff
@@ -0,0 +1,112 @@
...
+        folioInstances: testData.folioInstances,
+        location,
+      });
+    });
+    OtherSettings.setOtherSettingsViaApi({ titleLevelRequestsFeatureEnabled: true });
+    cy.createTempUser([
+      Permissions.uiRequestsCreate.gui,
+      Permissions.uiRequestsView.gui,
+      Permissions.uiRequestsEdit.gui,
+      Permissions.requestsAll.gui,
```
Comment (dnovikau) [inline cypress/e2e/circulation-log/verify-user-barcode-shown-in-request-actions.cy.js:41]: Please remove 'Permissions.requestsAll.gui,' permission. It's not mentioned in testcase.

**Review by dnovikau — CHANGES_REQUESTED** [review]

File: cypress/e2e/circulation-log/verify-user-barcode-shown-in-request-actions.cy.js (hunk 3 above)
Comment (victoriakor) [reply cypress/e2e/circulation-log/verify-user-barcode-shown-in-request-actions.cy.js]: done

File: cypress/e2e/circulation-log/verify-user-barcode-shown-in-request-actions.cy.js (hunk 1 above)
Comment (victoriakor) [reply cypress/e2e/circulation-log/verify-user-barcode-shown-in-request-actions.cy.js]: done

**Review by re-jas — APPROVED** [review]

**Pull request merged**